app/models/mandataris-status-code.js

```diff
@@ -3,4 +3,8 @@ import Model, { attr } from '@ember-data/model';
 export default class MandatarisStatusCodeModel extends Model {
   @attr label;
   @attr uri;
+
+  get isVerhinderd() {
```

Comment:
This might give some issues with titelvoerend, which is the burgemeester variant of verhinderd.

Reply:
Shouldn't we also check the concept uri instead of trying to match the label?

Reply:
Updated it to check on URI of verhinderd and titelvoerend